src/rshim_pcie.c

```diff
@@ -192,7 +191,7 @@ typedef struct {
 int bf3_rshim_pcie_chan_map[] = {
```

Comment:
BTW: Should this array be "static const int bf3_rshim_pcie_chan_map[]" ?

Reply:
Done. Updated version 2.